src/sorald.ts

```diff
@@ -130,7 +130,7 @@ function parseRepairedViolations(repairData: RepairData): string[] {
  * @param violationSpec - A violation specifier
  * @returns A closed range with the start and end lines of the violation
```

Comment:
Now you return a left-inclusive and right-exclusive range. :)

Reply:
Ah, that implementation is actually off-by-one now. The violation specifier is inclusive at both ends.

Reply:
In that case, let's merge this?